package.json

```diff
@@ -68,7 +68,8 @@
     "karma-jasmine-html-reporter": "^1.5.0",
     "protractor": "~7.0.0",
     "ts-node": "~9.0.0",
-    "typescript": "~4.2.4"
+    "typescript": "~4.2.4",
+    "webpack": "^5.43.0"
```

Comment:
Why do we need to add `webpack`?

Reply:
Because it's necessary to launch correctly the application, without that package, the ionic-app doesn't start.
I've seen this problem also with Jan-Vincent (jan-vincent.hoelzle@capgemini.com) and he suggest me to add that package to the package.json.

Reply:
After a second analysis the "webpack"  it's not necessary, o I've removed it.